src/components/AnonymousChat/child/ChannelFollowButton.tsx

```diff
@@ -38,30 +45,30 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center'
   },
-  followBtn: {
+  followBtn: (isAnonymousTab) => ({
     paddingVertical: dimen.normalizeDimen(9.5),
     paddingHorizontal: dimen.normalizeDimen(14),
     borderRadius: 8,
-    backgroundColor: colors.darkBlue
-  },
-  followingBtn: {
+    backgroundColor: isAnonymousTab ? COLORS.holyTosca : COLORS.blue
+  }),
+  followingBtn: (isAnonymousTab) => ({
     paddingVertical: dimen.normalizeDimen(9.5),
     paddingHorizontal: dimen.normalizeDimen(4.5),
     borderWidth: 1,
-    borderColor: colors.darkBlue,
+    borderColor: isAnonymousTab ? COLORS.holyTosca : COLORS.blue,
     borderRadius: 8
-  },
+  }),
   textFollowBtn: {
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 12
   },
   textFollow: {
-    color: colors.white
+    color: COLORS.white
   },
-  textFollowing: {
-    color: colors.darkBlue
-  }
+  textFollowing: (isAnonymousTab) => ({
+    color: isAnonymousTab ? COLORS.holyTosca : COLORS.blue
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles for `followBtn`, `followingBtn`, and `textFollowing` are now functions that take `isAnonymousTab` as a parameter and return an object. This allows for dynamic styling based on whether the tab is anonymous or not. The color values have been updated to use constants from the theme file, which improves maintainability.

```diff
  followBtn: (isAnonymousTab) => ({
     paddingVertical: dimen.normalizeDimen(9.5),
     paddingHorizontal: dimen.normalizeDimen(14),
     borderRadius: 8,
-    backgroundColor: colors.darkBlue
+    backgroundColor: isAnonymousTab ? COLORS.holyTosca : COLORS.blue
  }),
  followingBtn: (isAnonymousTab) => ({
     paddingVertical: dimen.normalizeDimen(9.5),
     paddingHorizontal: dimen.normalizeDimen(4.5),
     borderWidth: 1,
-    borderColor: colors.darkBlue,
+    borderColor: isAnonymousTab ? COLORS.holyTosca : COLORS.blue,
     borderRadius: 8
  }),
   textFollowBtn: {
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: 12
   },
   textFollow: {
-   color: colors.white
+   color: COLORS.white
   },
  textFollowing: (isAnonymousTab) => ({
-   color: colors.darkBlue
+   color: isAnonymousTab ? COLORS.holyTosca : COLORS.blue
  })
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->